assets/js/modules/tagmanager/setup.js

```diff
@@ -130,7 +130,23 @@ class TagmanagerSetup extends Component {
 				accountId: selectedAccount,
 			};
 
-			let responseData = await data.get( 'modules', 'tagmanager', 'list-accounts', queryArgs );
+			let errorCode    = false;
+			let errorMsg     = '';
+			let responseData = await data.get( 'modules', 'tagmanager', 'list-accounts', queryArgs, false );
```

Comment:
Is there any possibility of an error being thrown in this request?

Reply:
yes, it will throw error if no accounts found, state will be set in `catch( err )`

Reply:
Ah ok, I didn't see the surrounding try/catch in the diff, I should have looked first. Thanks!